src/framework/json.hpp

```diff
@@ -256,6 +313,173 @@ void std::to_json(json_t &js, const std::map<uint64_t, T1, T2> &map) {
   }
 }
 
+template <typename T>
+json_t numpy_to_json_1d(py::array_t<T, py::array::c_style> arr) {
+    py::buffer_info buf = arr.request();
+    if (buf.ndim != 1) {
+        throw std::runtime_error("Number of dims must be 1");
+    }
+
+    T *ptr = (T *) buf.ptr;
+    size_t D0 = buf.shape[0];
+
+    //std::cout << "1-d conversion: " << X << std::endl;
+
+    std::vector<T> tbr;
```

Comment:
What does `tbr` means? Can we change the name to something more descriptive?

Reply:
'to be returned' (sorry, I think that's a personal thing) - equivalent to 'ret' and 'out' (same file, to/from_json for complexes) - I'll change to 'ret'/'out' to be consistent and add a comment

Reply:
addressed in https://github.com/Qiskit/qiskit-aer/pull/399/commits/988a8e668e0ab77ce8c9b9d402d8fc889d3e0d52 (I just added an explanation where it is used because any name 'ret', 'out' would be equally non-descript)